content/community/tools-jsx.md

```diff
@@ -1,53 +1,56 @@
 
-## Build Tools {#build-tools}
+## أدوات البناء {#build-tools}
 
-* **[Create React App](https://github.com/facebookincubator/create-react-app):** An **officially supported** way to create React apps with no configuration.
-* **[nwb](https://github.com/insin/nwb)**: A toolkit for React, Preact & Inferno apps, React libraries and other npm modules for the web, with no configuration (until you need it)
-* **[Neutrino](https://neutrino.js.org/)**: Create and build modern JavaScript applications with zero initial configuration. Neutrino combines the power of webpack with the simplicity of presets.
-* **[ESLint](https://eslint.org/):** A pluggable JavaScript linter that natively supports JSX syntax. Be sure to download [eslint-plugin-react](https://npmjs.com/package/eslint-plugin-react) for React-specific rules.
-* **[Structor](https://www.npmjs.com/package/structor):** This tool is a user interface builder for node.js Web applications with React UI. Structor replaces the now deprecated React UI Builder. Watch [Structor Video Tutorials](https://youtu.be/z96xYa51EWI?list=PLAcaUOtEwjoR_U6eE2HQEXwkefeVESix1)
-* **[react-jsx](https://github.com/bigpipe/react-jsx):** Compile and use JSX as stand-alone templates that can run server-side and client side!
-* **[cjsx-codemod](https://github.com/jsdf/cjsx-codemod):** Write JSX code within Coffeescript!
-* **[ReactScript](https://github.com/1j01/react-script):** Write React code within Coffeescript without JSX!
+* **[Create React App](https://github.com/facebookincubator/create-react-app):** طريقة **مدعومة رسميًا** لإنشاء تطبيقات React بدون إعدادات.
+* **[nwb](https://github.com/insin/nwb)**: عُدّة لـReact و Preact و تطيبقات Inferno و مكتبات React و وحدات npm أخرى للويب وبدون إعدادات (إلى أن أنت تحتاجها).
+* **[Neutrino](https://neutrino.js.org/)**: إنشاء وبناء تطبيقات JavaScript متقدمة بدون إعدادات أولية. يجمع Neutrino بين قوة webpack مع بساطة المسبقة (presets).
+* **[ESLint](https://eslint.org/):** دقق JavaScript قابل للإضافة يدعم اصلا صيغة JSX. تأكد من تحميل [eslint-plugin-react](https://npmjs.com/package/eslint-plugin-react) لقواعد React-specific.
+* **[Structor](https://www.npmjs.com/package/structor):** هذه الأداة  لواجهة لمسخدم لبناء تطبيقات الويب بـNode.js مع React UI. Structor استبدلت بـReact UI Builder المهملة. شاهد [الفيديو التعليمي لـStructor](https://youtu.be/z96xYa51EWI?list=PLAcaUOtEwjoR_U6eE2HQEXwkefeVESix1).
+* **[react-jsx](https://github.com/bigpipe/react-jsx):** إعادة الصياغة وإستعمال JSX كنموذج مستقل يمكن إستعماله في جهة الخادم أو المستخدم!
+* **[cjsx-codemod](https://github.com/jsdf/cjsx-codemod):** كتابة الشفرة JSX من خلال Coffeescript!
+* **[ReactScript](https://github.com/1j01/react-script):** كتابة الشفرة React من خلال Coffeescript وبدون JSX!
 * **[jsxhint](https://npmjs.org/package/jsxhint):** [JSHint](http://jshint.com/) (linting) support. (JSX compilation doesn't affect line numbers so lint can also be run directly on the compiled JS.)
```

Comment:
Please review this:
![Screenshot from 2019-08-10 13-11-48](https://user-images.githubusercontent.com/9090674/62821678-87c3d400-bb70-11e9-81de-3510eb6d3df3.png)

Reply:
💚